### PR TITLE
readme: list (in)compatibility with specific controller features

### LINKED ISSUE #207: Document compatibility with the Absolute Accuracy Compensation feature
Context: #206.

Regardless of whether we can find a way to make MotoROS2 compatible with it, the documentation should be clear about whether or not MotoROS2 *is* compatible.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
Fixes #207.

I figured we might as well create a dedicated section as I expect we'll discover more incompatibilities, and the [Known issues & limitations](https://github.com/Yaskawa-Global/motoros2#known-issues--limitations) section was getting a bit long.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The following sections document how to download, install, configure, use and tro
 - [General Requirements](#general-requirements)
   - [Checking the system software version](#checking-the-system-software-version)
   - [Checking MotoPlus configuration](#checking-motoplus-configuration)
+- [Option compatibility](#option-compatibility)
```
Comment (gavanderhoorn) [inline README.md:53]: not sure about the title of the section.

I alternated between *option*, *feature* and some other things I don't remember.

@ted-miller: would there be any Yaskawa-standard terminology we could use for this?

**Comment by gavanderhoorn** [comment]
I'll rebase this on #209 to get rid of the `mdlint` errors.

File: README.md (hunk 1 above)
Comment (ted-miller) [reply README.md:53]: `Option Function` is the official term

**Comment by gavanderhoorn** [comment]
CI is happy again.

`mdlint` did actually flag one instance of using a different *strong style* correctly. Fixed it.

**Review by ted-miller — APPROVED** [review]

**Pull request merged**